layouts/partials/ecosystem/registry/entry.html

```diff
@@ -141,6 +141,19 @@ <h4 class="alert-heading">Deprecated</h4>
                     {{ end -}}
                     {{- .description | markdownify -}}
                 </div>
+
+                <!-- New Tags section -->
+                <div class="tags mb-2">
+                    {{ if .tags }}
+                        <div>
+                            {{ range .tags }}
+                                {{ $tag := . }} <!-- Store the tag in a variable -->
+                                <a href="/search?query={{ $tag | urlize }}" class="badge bg-light me-1">{{ $tag }}</a>
```

Comment:
```suggestion
                                <a href="/search?query={{ $tag | urlize }}" class="badge bg-light me-1">{{ $tag }}</a>
```

This sends the query to the page wide search, not the registry, the URL you are looking for is

```
/ecosystem/registry/?s=<tag>
```

Reply:
@svrnm Please kindly review if it sends to the registry search. Thank you